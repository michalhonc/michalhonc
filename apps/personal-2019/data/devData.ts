export const INSTAGRAM_DEV_DATA = [
  {
    big: "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/sh0.08/e35/s640x640/95357962_715605245852730_8234925818225125379_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=106&_nc_ohc=AfWzNBqXrv0AX-HnBv_&oh=c488e6434d184860467beb23140088c9&oe=5ED3E76B",
    small:
      "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/e15/s320x320/95357962_715605245852730_8234925818225125379_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=106&_nc_ohc=AfWzNBqXrv0AX-HnBv_&oh=ee044c803fefdd5f2e647b400d746cfd&oe=5ED3F51D",
    url: "https://instagram.com/p/B_pkUXWl9ll",
    caption:
      "Redesign of Inkscape. The vector based editor.‚Å†\n‚Å†\n#app #design #clean #color #creativity #designer #ui #uidesign #ux #website #webui #designapp #inspiration #designinspiration #dailyui #dailyinspiration #userinterface #graphicdesign #mobile #code #web #digitaldesign #mockup  #figma #sketch #adobe #photoshop #inkscape #vector #redesign",
    likes: 8,
    comments: 0,
    id: "2299528813888395621",
  },
  {
    big: "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/sh0.08/e35/s640x640/95993828_2646409262132470_7336183499360016298_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=110&_nc_ohc=T6G4TrDEr60AX_f_5rq&oh=898111d719a30fabd6199574c3022853&oe=5ED6DBF5",
    small:
      "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/e35/s320x320/95993828_2646409262132470_7336183499360016298_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=110&_nc_ohc=T6G4TrDEr60AX_f_5rq&oh=18996d0e2eb24aba17c70fcef714b224&oe=5ED6630A",
    url: "https://instagram.com/p/B_o_GQqBfy7",
    caption:
      "9 Design Idea Generators for free\n\nLearn design effortlessly with these tools. You dont have to think about what to create\n\n#app #design #clean #color #creativity #designer #ui #uidesign #ux #website #webui #designapp #inspiration #designinspiration #dailyui #dailyinspiration #userinterface #graphicdesign #mobile #code #web #digitaldesign #mockup  #figma #sketch #adobe #photoshop #idea #ideas",
    likes: 6,
    comments: 0,
    id: "2299365116904602811",
  },
  {
    big: "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/sh0.08/e35/s640x640/95260378_170508607750106_2829547284297747694_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=102&_nc_ohc=U5aFZiajCM8AX9ou-ks&oh=b421034da35b7d31b3708ad5ccd8d363&oe=5ED61950",
    small:
      "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/e15/s320x320/95260378_170508607750106_2829547284297747694_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=102&_nc_ohc=U5aFZiajCM8AX9ou-ks&oh=e148610acba884895a75b47bffc172c8&oe=5ED7403A",
    url: "https://instagram.com/p/B_m_iCUguFJ",
    caption:
      "Floweree‚Å†\nDesign sketch of florist‚Å†\n‚Å†\n#app #design #clean #color #creativity #designer #ui #uidesign #ux #website #webui #designapp #inspiration #designinspiration #dailyui #dailyinspiration #userinterface #graphicdesign #mobile #code #web #digitaldesign #mockup  #figma #sketch #adobe #photoshop #florist #flower #flowers",
    likes: 12,
    comments: 0,
    id: "2298804075703230793",
  },
  {
    big: "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/sh0.08/e35/s640x640/94911261_617608922159714_6405634009694963018_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=102&_nc_ohc=BQU83wpS8mIAX_6aaxj&oh=67c1d0dce09b5e4867f6ad8b644ddd6c&oe=5ED5B904",
    small:
      "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/e15/s320x320/94911261_617608922159714_6405634009694963018_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=102&_nc_ohc=BQU83wpS8mIAX_6aaxj&oh=1c8ada4ace68a23ea9ab3f5c95910934&oe=5ED40DF6",
    url: "https://instagram.com/p/B_mW4F3gUY3",
    caption:
      "Radnika Next‚Å†\n‚Å†\nPopular modern sans typeface that can be used anywhere‚Å†\n‚Å†\n#app #design #clean #color #creativity #designer #ui #uidesign #ux #website #webui #designapp #inspiration #designinspiration #dailyui #dailyinspiration #userinterface #graphicdesign #mobile #typeface #web #sans #mockup  #figma #sketch #adobe #photoshop #font #radnika #typography",
    likes: 9,
    comments: 1,
    id: "2298625271433086519",
  },
  {
    big: "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/sh0.08/e35/s640x640/94448120_3188642834488564_3431111247580652806_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=104&_nc_ohc=khugG1QxutoAX9AOhCD&oh=eca4f69dbf63ecf75a2c2379efeb1955&oe=5ED57C1B",
    small:
      "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/e15/s320x320/94448120_3188642834488564_3431111247580652806_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=104&_nc_ohc=khugG1QxutoAX9AOhCD&oh=492ce685632ce333fab4e7eb14dd5af6&oe=5ED49CE0",
    url: "https://instagram.com/p/B_kavmcnG67",
    caption:
      "#app #design #clean #color #creativity #designer #ui #uidesign #ux #website #webui #designapp #inspiration #designinspiration #dailyui #dailyinspiration #userinterface #graphicdesign #mobile #code #web #digitaldesign #mockup  #figma #sketch #adobe #photoshop #painter #artist #abstract",
    likes: 16,
    comments: 0,
    id: "2298079330172694203",
  },
  {
    big: "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/sh0.08/e35/s640x640/95173308_240982723648747_577521512543216677_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=107&_nc_ohc=-Uc8a3oIqFMAX86hsxl&oh=2dbd38aa17dcc4b0fb8981352f03a535&oe=5ED45BED",
    small:
      "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/e15/s320x320/95173308_240982723648747_577521512543216677_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=107&_nc_ohc=-Uc8a3oIqFMAX86hsxl&oh=07d560ae2a392c79d4464a28027aabac&oe=5ED77BFF",
    url: "https://instagram.com/p/B_jxhaGAJyu",
    caption:
      "Pressed button in neumorphism design concept.‚Å†\n‚Å†\nUse it in your designs!‚Å†\n‚Å†\n#app #design #clean #color #creativity #designer #ui #uidesign #ux #website #webui #designapp #inspiration #designinspiration #dailyui #dailyinspiration #userinterface #graphicdesign #mobile #code #web #digitaldesign #mockup  #figma #sketch #adobe #photoshop #neumorphism #button",
    likes: 17,
    comments: 0,
    id: "2297898034928852142",
  },
  {
    big: "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/sh0.08/e35/s640x640/94733384_546690222933677_7757530875386802573_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=105&_nc_ohc=5R-rS4bm-JYAX-zJuuW&oh=5a1fc1a93f3df25871b7299cc3dd05e6&oe=5ED78C33",
    small:
      "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/e15/s320x320/94733384_546690222933677_7757530875386802573_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=105&_nc_ohc=5R-rS4bm-JYAX-zJuuW&oh=a98b8964c6a4b81f3ae04b3fd87dddfc&oe=5ED5C645",
    url: "https://instagram.com/p/B_h18VSHHKc",
    caption:
      "Create spacing system for your design with Fibonacci sequence. One of many ways to set your spacing system‚Å†\n‚Å†\n#app #design #clean #color #creativity #designer #ui #uidesign #ux #website #webui #designapp #inspiration #designinspiration #dailyui #dailyinspiration #userinterface #graphicdesign #mobile #code #web #digitaldesign  #figma #sketch #adobe #photoshop #spacing #spacingsystem #system #fibonacci",
    likes: 8,
    comments: 0,
    id: "2297354527421788828",
  },
  {
    big: "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/sh0.08/e35/s640x640/94559667_154040786109956_8589982848374977447_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=108&_nc_ohc=vb6NjU8fg1gAX_cqDef&oh=0ea93f3a5455524852b77c493a98257f&oe=5ED58323",
    small:
      "https://scontent-prg1-1.cdninstagram.com/v/t51.2885-15/e35/s320x320/94559667_154040786109956_8589982848374977447_n.jpg?_nc_ht=scontent-prg1-1.cdninstagram.com&_nc_cat=108&_nc_ohc=vb6NjU8fg1gAX_cqDef&oh=806a84ed617bd6579557cd08430fb28a&oe=5ED413F6",
    url: "https://instagram.com/p/B_hTKWzhgWS",
    caption:
      "Use Pureref in your design process. You can organize your inspiration and speed up your creative process with it.\n\nPuRef is a stand-alone program for Mac, Windows and Linux that keeps track of your images. Whether your are gathering inspiration, making mood boards or need reference images for your painting or 3D model, PureRef is there so you can focus on creating.\n\n#app #design #clean #color #creativity #designer #ui #uidesign #ux #website #webui #designapp #inspiration #designinspiration #dailyui #dailyinspiration #userinterface #graphicdesign #mobile #code #web #digitaldesign #mockup  #figma #sketch #adobe #photoshop #pureref #tip #howto",
    likes: 10,
    comments: 0,
    id: "2297201559500883346",
  },
];

export const TWITTER_DEV_DATA = [
  {
    createdAt: "Fri Apr 24 07:20:51 +0000 2020",
    text: "RT @mathias: üí° In source code, stop using the old rgb()/hsl() CSS color syntax with commas.\n\nGet used to the modern comma-free CSS color sy‚Ä¶",
    small: null,
    thumb: null,
    id: 1253584675546837000,
    url: "https://twitter.com/Michalhonc/status/1253584675546836993",
  },
  {
    createdAt: "Thu Apr 23 15:38:20 +0000 2020",
    text: "Did you know that you can debug DevTools within DevTools? #javascript #webdev #hottip #devtools @ChromeDevTools https://t.co/Nnk63TD3Cq",
    small: "http://pbs.twimg.com/tweet_video_thumb/EWTIX0dWkAMTNrG.jpg:small",
    thumb: "http://pbs.twimg.com/tweet_video_thumb/EWTIX0dWkAMTNrG.jpg:thumb",
    id: 1253347481313857500,
    url: "https://twitter.com/Michalhonc/status/1253347481313857536",
  },
  {
    createdAt: "Sat Apr 18 10:43:19 +0000 2020",
    text: "Did you know that you can simply drag&amp;drop elements in \n@ChromeDevTools and @FirefoxDevTools? Useful in sorting tex‚Ä¶ https://t.co/X6ADUEvb8V",
    small: null,
    thumb: null,
    id: 1251461299084636200,
    url: "https://twitter.com/Michalhonc/status/1251461299084636163",
  },
  {
    createdAt: "Sat Apr 18 05:32:31 +0000 2020",
    text: "What if jQuery was released today? #jquery #redesign #figma #uidesign #webdesign https://t.co/njtMnoUbPA",
    small: null,
    thumb: null,
    id: 1251383083678470100,
    url: "https://twitter.com/Michalhonc/status/1251383083678470145",
  },
  {
    createdAt: "Tue Apr 14 16:48:09 +0000 2020",
    text: "Run `npm run` to list all scripts within NPM project! #webdevelopment #npm #reactjs #dev https://t.co/FBitNUoT5B",
    small: "http://pbs.twimg.com/media/EVlCc_aWAAE5MPI.jpg:small",
    thumb: "http://pbs.twimg.com/media/EVlCc_aWAAE5MPI.jpg:thumb",
    id: 1250103559854661600,
    url: "https://twitter.com/Michalhonc/status/1250103559854661634",
  },
  {
    createdAt: "Sun Apr 05 13:19:46 +0000 2020",
    text: "Add `document.designMode = 'on'` to your site for realtime updates to text on screen. Great for UI designers and co‚Ä¶ https://t.co/2JFvJ9wzky",
    small: null,
    thumb: null,
    id: 1246789629547892700,
    url: "https://twitter.com/Michalhonc/status/1246789629547892737",
  },
];
