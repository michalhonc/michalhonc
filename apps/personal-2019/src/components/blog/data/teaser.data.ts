export const Query = {
  allMarkdownRemark: {
    edges: [
      {
        node: {
          frontmatter: {
            date: "2020-03-31",
            language: "en",
            description:
              "Canny tricks that won't make your React app better but will make your dev process easier.",
            path: "/blog/5-canny-tricks-for-react-devs",
            title: "5 Canny little tricks for React devs",
            category: "react",
            thumbnail: {
              childImageSharp: {
                fluid: {
                  src: "/static/4e57a90ee8c2f36cfe0eba16def56413/3ee52/5_canny_react_tricks.png",
                  aspectRatio: 1.4184397163120568,
                  srcSet:
                    "/static/4e57a90ee8c2f36cfe0eba16def56413/69585/5_canny_react_tricks.png 200w,\n/static/4e57a90ee8c2f36cfe0eba16def56413/497c6/5_canny_react_tricks.png 400w,\n/static/4e57a90ee8c2f36cfe0eba16def56413/3ee52/5_canny_react_tricks.png 704w",
                  sizes: "(max-width: 704px) 100vw, 704px",
                },
              },
            },
          },
        },
      },
      {
        node: {
          frontmatter: {
            date: "2020-03-31",
            language: "cs",
            description:
              "Triky, které Vaši aplikaci nezlepší, ale usnadní vám proces vývoje",
            path: "/blog/5-triku-pro-vyvojare-reactu",
            title: "5 triků pro vývojáře Reactu",
            category: "react",
            thumbnail: {
              childImageSharp: {
                fluid: {
                  src: "/static/4e57a90ee8c2f36cfe0eba16def56413/3ee52/5_canny_react_tricks.png",
                  aspectRatio: 1.4184397163120568,
                  srcSet:
                    "/static/4e57a90ee8c2f36cfe0eba16def56413/69585/5_canny_react_tricks.png 200w,\n/static/4e57a90ee8c2f36cfe0eba16def56413/497c6/5_canny_react_tricks.png 400w,\n/static/4e57a90ee8c2f36cfe0eba16def56413/3ee52/5_canny_react_tricks.png 704w",
                  sizes: "(max-width: 704px) 100vw, 704px",
                },
              },
            },
          },
        },
      },
      {
        node: {
          frontmatter: {
            date: "2020-02-25",
            language: "cs",
            description:
              "Nepřicházejte kvůli AdBlocku o peníze z reklamy! Zjištěte 7 způsobů jak se proti AdBlocku můžete bránit.",
            path: "/blog/7-zpusobu-jak-obejit-adblock",
            title: "7 způsobů jak obejít AdBlock",
            category: "reklama",
            thumbnail: {
              childImageSharp: {
                fluid: {
                  src: "/static/d96fb15f308d10e6d5035097232fbd35/46604/adblock_thumbnail.png",
                  aspectRatio: 1.492537313432836,
                  srcSet:
                    "/static/d96fb15f308d10e6d5035097232fbd35/69585/adblock_thumbnail.png 200w,\n/static/d96fb15f308d10e6d5035097232fbd35/497c6/adblock_thumbnail.png 400w,\n/static/d96fb15f308d10e6d5035097232fbd35/46604/adblock_thumbnail.png 500w",
                  sizes: "(max-width: 500px) 100vw, 500px",
                },
              },
            },
          },
        },
      },
      {
        node: {
          frontmatter: {
            date: "2020-02-25",
            language: "en",
            description:
              "Don't lose advertising money because of AdBlock! Find out 7 ways you can defend yourself against AdBlock.",
            path: "/blog/7-ways-to-bypass-adblock",
            title: "7 ways to bypass AdBlock",
            category: "ads",
            thumbnail: {
              childImageSharp: {
                fluid: {
                  src: "/static/d96fb15f308d10e6d5035097232fbd35/46604/adblock_thumbnail.png",
                  aspectRatio: 1.492537313432836,
                  srcSet:
                    "/static/d96fb15f308d10e6d5035097232fbd35/69585/adblock_thumbnail.png 200w,\n/static/d96fb15f308d10e6d5035097232fbd35/497c6/adblock_thumbnail.png 400w,\n/static/d96fb15f308d10e6d5035097232fbd35/46604/adblock_thumbnail.png 500w",
                  sizes: "(max-width: 500px) 100vw, 500px",
                },
              },
            },
          },
        },
      },
      {
        node: {
          frontmatter: {
            date: "2020-04-08",
            language: "en",
            description:
              "In this tutorial I'll show you how to set up professional development process with help of continuous deployment.",
            path: "/blog/continuous-deployment-for-free",
            title:
              "Continuous development for FREE with Github, Travis CI and Heroku",
            category: "continuous deployment",
            thumbnail: {
              childImageSharp: {
                fluid: {
                  src: "/static/f53b530a16d87556855e6e551c169f83/f0c16/travis_thumbnail.png",
                  aspectRatio: 1.3605442176870748,
                  srcSet:
                    "/static/f53b530a16d87556855e6e551c169f83/69585/travis_thumbnail.png 200w,\n/static/f53b530a16d87556855e6e551c169f83/f0c16/travis_thumbnail.png 387w",
                  sizes: "(max-width: 387px) 100vw, 387px",
                },
              },
            },
          },
        },
      },
      {
        node: {
          frontmatter: {
            date: "2020-04-08",
            language: "cs",
            description:
              "V tomto tutorialu vám ukáži, jak nastavit profesionální vývojové prostřední s pomocí continuous deployment.",
            path: "/blog/continuous-deployment-zdarma",
            title: "Continuous development zdarma s Github, Travis CI a Heroku",
            category: "continuous deployment",
            thumbnail: {
              childImageSharp: {
                fluid: {
                  src: "/static/f53b530a16d87556855e6e551c169f83/f0c16/travis_thumbnail.png",
                  aspectRatio: 1.3605442176870748,
                  srcSet:
                    "/static/f53b530a16d87556855e6e551c169f83/69585/travis_thumbnail.png 200w,\n/static/f53b530a16d87556855e6e551c169f83/f0c16/travis_thumbnail.png 387w",
                  sizes: "(max-width: 387px) 100vw, 387px",
                },
              },
            },
          },
        },
      },
    ],
  },
};
