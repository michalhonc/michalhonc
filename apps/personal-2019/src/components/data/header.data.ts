export const HeaderQuery = {
  background: {
    childImageSharp: {
      fluid: {
        src: "/static/45779372c1991322e2cf8821eb9b133f/74fd5/header-background.jpg",
        aspectRatio: 1.3333333333333333,
        srcSet:
          "/static/45779372c1991322e2cf8821eb9b133f/e3903/header-background.jpg 480w,\n/static/45779372c1991322e2cf8821eb9b133f/b31d1/header-background.jpg 960w,\n/static/45779372c1991322e2cf8821eb9b133f/74fd5/header-background.jpg 1920w,\n/static/45779372c1991322e2cf8821eb9b133f/cb67e/header-background.jpg 2483w",
        sizes: "(max-width: 1920px) 100vw, 1920px",
      },
    },
  },
  foreground: {
    childImageSharp: {
      fluid: {
        src: "/static/96b4dfdae8e006ed6b85bfb9aadd902b/0e6e2/header-foreground.png",
        aspectRatio: 1.3333333333333333,
        srcSet:
          "/static/96b4dfdae8e006ed6b85bfb9aadd902b/b460a/header-foreground.png 480w,\n/static/96b4dfdae8e006ed6b85bfb9aadd902b/6e898/header-foreground.png 960w,\n/static/96b4dfdae8e006ed6b85bfb9aadd902b/0e6e2/header-foreground.png 1920w,\n/static/96b4dfdae8e006ed6b85bfb9aadd902b/4e88d/header-foreground.png 2880w,\n/static/96b4dfdae8e006ed6b85bfb9aadd902b/049dd/header-foreground.png 3309w",
        sizes: "(max-width: 1920px) 100vw, 1920px",
      },
    },
  },
};
