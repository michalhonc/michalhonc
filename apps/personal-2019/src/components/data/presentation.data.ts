export const PresentationQuery = {
  mockup: {
    childImageSharp: {
      fluid: {
        aspectRatio: 0.7092198581560284,
        src: "/static/ea55a42170bb63d820b1b81f9df1bf59/3891b/mockup.png",
        srcSet:
          "/static/ea55a42170bb63d820b1b81f9df1bf59/3ba9f/mockup.png 100w,\n/static/ea55a42170bb63d820b1b81f9df1bf59/8ac63/mockup.png 200w,\n/static/ea55a42170bb63d820b1b81f9df1bf59/3891b/mockup.png 400w,\n/static/ea55a42170bb63d820b1b81f9df1bf59/8c332/mockup.png 600w,\n/static/ea55a42170bb63d820b1b81f9df1bf59/bc8e0/mockup.png 800w,\n/static/ea55a42170bb63d820b1b81f9df1bf59/8d364/mockup.png 1500w",
        sizes: "(max-width: 400px) 100vw, 400px",
      },
    },
  },
};
