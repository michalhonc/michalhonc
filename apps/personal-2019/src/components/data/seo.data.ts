export const SeoQuery = {
  site: {
    siteMetadata: {
      author: "Michal Honc",
      description:
        "První konzultace ZDARMA! Vytvořím Vám rychlé a moderní webové stránky, se kterýma Vy nebo Váš produkt uspějete. Sjednejte si konzultaci zdarma během jedné minuty",
      title: "Michal Honc",
    },
  },
};
