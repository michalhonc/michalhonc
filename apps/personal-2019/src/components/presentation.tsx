import * as React from "react";
import { useStaticQuery, graphql } from "gatsby";
import Img from "gatsby-image";
import { useIntl } from "gatsby-plugin-intl";

import Lines from "../images/lines.svg";
import Dots from "../images/dots.svg";

import { ReservationConsultation } from "./reservationConsultation";
const Query = graphql`
  {
    mockup: file(relativePath: { eq: "mockup.png" }) {
      childImageSharp {
        fluid(maxWidth: 400, quality: 100) {
          ...GatsbyImageSharpFluid_withWebp
        }
      }
    }
  }
`;

export const Presentation = () => {
  const intl = useIntl();
  const { mockup } = useStaticQuery(Query);

  return (
    <section className="Container Presentation">
      <h2
        id="presentation"
        dangerouslySetInnerHTML={{
          __html: intl.formatMessage({ id: "presentation_title" }),
        }}
      />
      <div className="Presentation-container">
        <div className="Presentation-photo">
          <Img fluid={mockup.childImageSharp.fluid} />
          <Lines className="Presentation-lines" />
        </div>
        <div className="Presentation-content">
          <ul>
            <li
              dangerouslySetInnerHTML={{
                __html: intl.formatMessage({ id: "presentation_text_1" }),
              }}
            />
            <li
              dangerouslySetInnerHTML={{
                __html: intl.formatMessage({ id: "presentation_text_2" }),
              }}
            />
            <li
              dangerouslySetInnerHTML={{
                __html: intl.formatMessage({ id: "presentation_text_3" }),
              }}
            />
          </ul>
          <ReservationConsultation />
        </div>
      </div>
    </section>
  );
};
