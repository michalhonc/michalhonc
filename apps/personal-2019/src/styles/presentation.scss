.Presentation {
	b {
		box-shadow: inset 0 -30px 0 -10px var(--c-alpha-light);
	}

	&-container {
		display: flex;

    @include respond(phone) {
      flex-direction: column;
      padding: 0 1rem;
    }
	}

	&-lines {
		position: absolute;
		top: 18%;
		left: 7%;
		width: 70%;
		z-index: -1;

    @include respond(phone) {
      top: 5%;
      left: 0;
      width: 90%;
    }
	}

  &-photo {
		position: relative;
		width: 80%;
		margin-right: 5rem;

    @include respond(phone) {
      margin: 0 auto;
      width: 50%;
    }
  }

  &-content {
		display: flex;
		flex-direction: column;

    & .ReservationConsultation {
      margin-top: 3rem;
      width: 32rem;

      @include respond(phone) {
        width: 70%;
        max-width: unset;
        margin: 2rem auto 0;
      }
    }

		ul {
			display: flex;
			flex-direction: column;
			padding-top: 3rem;

      @include respond(phone) {
        padding-top: 1.5rem;
      }

			li {
				font-size: 2.8rem;
				color: var(--c-beta);
				margin: 3rem 0;
				list-style: none;
				position: relative;
				padding-left: 5rem;

        @include respond(phone) {
          font-size: 1.8rem;
          margin: 1.5rem 0;
          padding-left: 3rem;
        }

				&::before {
					content: "";
					position: absolute;
					left: 0;
					top: 0.5rem;
					width: 2.5rem;
					height: 2.5rem;
					border-radius: 50%;
					border: 0.5rem solid var(--c-alpha);

          @include respond(phone) {
              width: 1.5rem;
              height: 1.5rem;
              border: 0.3rem solid var(--c-alpha);
          }
				}
			}
		}
  }
}
