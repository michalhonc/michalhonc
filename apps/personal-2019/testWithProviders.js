import React from "react";
import renderer from "react-test-renderer";
import { IntlContextProvider } from "gatsby-plugin-intl";

export const renderWithProviders = (component) =>
  renderer.create(
    <IntlContextProvider value={{ language: "en-US", routed: true }}>
      {component}
    </IntlContextProvider>
  );
