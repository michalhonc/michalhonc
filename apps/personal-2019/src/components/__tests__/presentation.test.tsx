import * as React from "react";
import * as Gatsby from "gatsby";
import renderer from "react-test-renderer";
import { renderWithProviders } from "../../../testWithProviders";
import { Presentation } from "../presentation";
import { PresentationQuery } from "../data/presentation.data";

describe("presentation", () => {
  const useStaticQuery = jest.spyOn(Gatsby, "useStaticQuery");
  useStaticQuery.mockImplementation(() => PresentationQuery);

  it("renders correctly", () => {
    const tree = renderWithProviders(<Presentation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
