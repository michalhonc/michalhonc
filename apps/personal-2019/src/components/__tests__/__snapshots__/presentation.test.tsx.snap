// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`presentation renders correctly 1`] = `
<section
  className="Container Presentation"
>
  <h2
    dangerouslySetInnerHTML={
      Object {
        "__html": "Vybuduji Vám <span class='underline'>kvalitní</span> webovou prezentaci",
      }
    }
    id="presentation"
  />
  <div
    className="Presentation-container"
  >
    <div
      className="Presentation-photo"
    >
      <div
        className=" gatsby-image-wrapper"
        style={
          Object {
            "overflow": "hidden",
            "position": "relative",
          }
        }
      >
        <div
          aria-hidden={true}
          style={
            Object {
              "paddingBottom": "141%",
              "width": "100%",
            }
          }
        />
        <picture>
          <source
            sizes="(max-width: 400px) 100vw, 400px"
            srcSet="/static/ea55a42170bb63d820b1b81f9df1bf59/3ba9f/mockup.png 100w,
/static/ea55a42170bb63d820b1b81f9df1bf59/8ac63/mockup.png 200w,
/static/ea55a42170bb63d820b1b81f9df1bf59/3891b/mockup.png 400w,
/static/ea55a42170bb63d820b1b81f9df1bf59/8c332/mockup.png 600w,
/static/ea55a42170bb63d820b1b81f9df1bf59/bc8e0/mockup.png 800w,
/static/ea55a42170bb63d820b1b81f9df1bf59/8d364/mockup.png 1500w"
          />
          <img
            alt=""
            loading="lazy"
            onLoad={[Function]}
            sizes="(max-width: 400px) 100vw, 400px"
            src="/static/ea55a42170bb63d820b1b81f9df1bf59/3891b/mockup.png"
            srcSet="/static/ea55a42170bb63d820b1b81f9df1bf59/3ba9f/mockup.png 100w,
/static/ea55a42170bb63d820b1b81f9df1bf59/8ac63/mockup.png 200w,
/static/ea55a42170bb63d820b1b81f9df1bf59/3891b/mockup.png 400w,
/static/ea55a42170bb63d820b1b81f9df1bf59/8c332/mockup.png 600w,
/static/ea55a42170bb63d820b1b81f9df1bf59/bc8e0/mockup.png 800w,
/static/ea55a42170bb63d820b1b81f9df1bf59/8d364/mockup.png 1500w"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "objectFit": "cover",
                "objectPosition": "center",
                "opacity": 0,
                "position": "absolute",
                "top": 0,
                "transition": "opacity 500ms",
                "width": "100%",
              }
            }
          />
        </picture>
        <noscript
          dangerouslySetInnerHTML={
            Object {
              "__html": "<picture><source srcset=\\"/static/ea55a42170bb63d820b1b81f9df1bf59/3ba9f/mockup.png 100w,
/static/ea55a42170bb63d820b1b81f9df1bf59/8ac63/mockup.png 200w,
/static/ea55a42170bb63d820b1b81f9df1bf59/3891b/mockup.png 400w,
/static/ea55a42170bb63d820b1b81f9df1bf59/8c332/mockup.png 600w,
/static/ea55a42170bb63d820b1b81f9df1bf59/bc8e0/mockup.png 800w,
/static/ea55a42170bb63d820b1b81f9df1bf59/8d364/mockup.png 1500w\\" sizes=\\"(max-width: 400px) 100vw, 400px\\" /><img loading=\\"lazy\\" sizes=\\"(max-width: 400px) 100vw, 400px\\" srcset=\\"/static/ea55a42170bb63d820b1b81f9df1bf59/3ba9f/mockup.png 100w,
/static/ea55a42170bb63d820b1b81f9df1bf59/8ac63/mockup.png 200w,
/static/ea55a42170bb63d820b1b81f9df1bf59/3891b/mockup.png 400w,
/static/ea55a42170bb63d820b1b81f9df1bf59/8c332/mockup.png 600w,
/static/ea55a42170bb63d820b1b81f9df1bf59/bc8e0/mockup.png 800w,
/static/ea55a42170bb63d820b1b81f9df1bf59/8d364/mockup.png 1500w\\" src=\\"/static/ea55a42170bb63d820b1b81f9df1bf59/3891b/mockup.png\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
            }
          }
        />
      </div>
      <test-file-stub
        className="Presentation-lines"
      />
    </div>
    <div
      className="Presentation-content"
    >
      <ul>
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Navrhuji webové stránky, které <b>výborně</b> prezentují Vás a Vaše produkty",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Vytvářím <b>3x</b> rychlejší stránky oproti průměru",
            }
          }
        />
        <li
          dangerouslySetInnerHTML={
            Object {
              "__html": "Mám rozsáhlé spektrum <b>zkušeností</b>. Od živnostníků až po českou internetovou jedničku.",
            }
          }
        />
      </ul>
      <button
        className="ReservationConsultation"
        onClick={[Function]}
        type="button"
      >
        Rezervovat konzultaci zdarma
      </button>
    </div>
  </div>
</section>
`;
